README.md

```diff
@@ -99,7 +105,16 @@ For details about building and testing DNF5, see the [Development environment se
 
 Translating
 -----------
-TBD
+
+Tranlating DNF5 from English to other langauges happens at
+[Fedora Weblate](https://translate.fedoraproject.org/projects/dnf5/)
+translation web site.
+
+The web site extracts new English messages from this repository. Users of the web
+site translate them there and the web site exports the finshed translations to
+[dnf5-l10n](https://github.com/rpm-software-management/dnf5-l10n) repository.
+where from the translation catalogs are copied into a DNF5 release tar ball on
```

Comment:
Is this a typo?
I think the sentence should start with a capital letter plus the `where from the ...` sentence structure seems odd to me.

Reply:
Perhaps something like: `The translation catalogs from the repository are copied into a DNF5 release tar ball on every DNF5 release.`?

Reply:
I'm sorry. I rewrote the paragraph few times and forgot to finish the last sentence.